redisvl/llmcache/base.py

```diff
@@ -26,22 +26,3 @@ def _refresh_ttl(self, key: str):
     def hash_input(self, prompt: str):
         """Hashes the input using SHA256."""
         return hashlib.sha256(prompt.encode("utf-8")).hexdigest()
-
```

Comment:
don't want to kep the decorator option?

Reply:
I'll explain, needs more work on the edge cases. I found too many ways to break it for me to be comfortable for a release with it.